Integrations/integration-OpsGenie.yml

```diff
@@ -38,8 +38,9 @@ script:
         users: 'users'
     };
 
-    var sendRequest = function(url) {
-        var requestUrl = baseURL + url;
+    var sendRequest = function(url, param) {
```

Comment:
you added the `param` parameter as second argument.
I checked all `sendRequest` calls, and it seems that you never pass the second argument... am I missing something? If not in use, please remove.

Reply:
you are right. had it FFU, but no need for now.